jaxlib/BUILD

```diff
@@ -121,6 +121,8 @@ pybind_extension(
     deps = [
         ":gpu_kernel_helpers",
         ":kernel_pybind11_helpers",
+        "@org_tensorflow//tensorflow/stream_executor/cuda:cublas_lib",
+        "@org_tensorflow//tensorflow/stream_executor/cuda:cudart_stub",
```

Comment:
For my understanding, what does changing these cuda deps do? Does this mean they're built + statically linked from the TF repo so users don't need to have these libs installed?

Reply:
These targets in `stream_executor` are actually little redirection libraries that either:
a) link the original library if `open_source_build` is not defined
b) link a stub that `dlopen()`s the library rather than linking it.

It is (b) that we want here: `manylinux2010` wheels must not link to anything but a whitelist of libraries.